src/PointwiseFunctions/AnalyticSolutions/Errors.hpp

```diff
@@ -0,0 +1,73 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Variables.hpp"
+#include "DataStructures/VariablesTag.hpp"
+#include "Domain/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Tags {
+/*!
+ * \brief Prefix indicating the difference between two values, such as betwen a
```

Comment:
Please adjust the documentation to reflect this now represents the error, not any difference